website/other/vimrc

```diff
@@ -0,0 +1,57 @@
+" A simple default configuration for beginner Vim usage.
+"
+" If you have any questions about this file, send and email to
+" edvth289@student.liu.se, made with <3 by LiTHe kod.
+
+" This is strictly for Vim, not the ancient Vi.
+set nocompatible
+
+" Keep the undo history, lets you close the buffer and
+" still undo.
+set undofile
```

Comment:
```suggestion
" Keep the undo history, lets you close the buffer and
" still undo.
if !isdirectory($HOME."/.vim")
    call mkdir($HOME."/.vim", "", 0770)
endif
if !isdirectory($HOME."/.vim/undo-dir")
    call mkdir($HOME."/.vim/undo-dir", "", 0700)
endif
set undodir=~/.vim/undo-dir
set undofile
```

https://vi.stackexchange.com/a/53

I don't know if this works on Windows, but I think it should.